app/modules/UtilityModule.py

```diff
@@ -0,0 +1,15 @@
+import json
+import subprocess
+from pymongo import MongoClient
+
+class Rescan(object):
+
+	def rescan(self):
+		# processMsg = subprocess.check_output(['python3','test.py']).decode('utf-8').split("\n")
+		# print(processMsg)
+		# print("THis is a test of the scan feature to be implemented")
+		# print(psutil.process_iter())
+		# for proc in psutil.process_iter():
+		# 	print(proc)
+		# return processMsg
+		subprocess.run(['python3','/opt/miflora-mqtt-daemon/miflora-mqtt-daemon.py'])
```

Comment:
`miflora-mqtt-daemon.py` just polls the sensors that are already connected, and doesn't reach out to find new sensors. As far as I know, this will run indefinitely. Do we really want to have multiple `miflora-mqtt-daemon.py` going at the same time?

Reply:
Removed scan button and functionality from code.